lib/absinthe/type/scalar.ex

```diff
@@ -19,13 +19,13 @@ defmodule Absinthe.Type.Scalar do
 
   ## Examples
 
-  Supporting a time format in ISOz format, using [Timex](http://hexdocs.pm/timex):
+  Supporting datetime in the ISO8601 format for UTC only, using [Timex](https://hexdocs.pm/timex):
 
   ```
-  scalar :time do
-    description "Time (in ISOz format)"
-    parse &Timex.DateFormat.parse(&1, "{ISOz}")
-    serialize &Timex.DateFormat.format!(&1, "{ISOz}")
+  scalar :isoz_datetime do
+    description "UTC only ISO8601 date time"
+    parse &Timex.parse(&1, "{ISO:Extended:Z}")
+    serialize &Timex.format!(&1, "{ISO:Extended:Z}")
```

Comment:
If we're updating this, we may as well switch this over to using the built in `DateTime` functions.

Reply:
@benwilson512 I also considered the same thing. But since `datetime` it is already implemented in [Absinthe.Type.Custom](https://hexdocs.pm/absinthe/Absinthe.Type.Custom.html) I tried to maintain it as it was originally created: for solving a custom scenario that is not already included with absinthe.

Absinthe's `datetime` will do DateTime.from_iso8601(value), meaning it is fully ISO8601 compliment. In the example above it actually only accepts ISO8601 in the UTC time-zone, hence the "UTC only" mention.
IE: Will accept `2022-10-17T23:27:45.076302Z` but not `2022-10-17T23:27:45.076302+1000`.

I don't want to encourage users to re-implement the datetime type. Maybe I should add a note or redirect users to `Absinthe.Type.Custom`?

Reply:
I've just made it clearer with f617cf40367bed55b4284ef97b2bb465d1c3841d
What do you think?